contrib/styles.scss

```diff
@@ -0,0 +1,298 @@
+}
+
+// Header
+// ============================================
+
+.container {
+  @include margin(null auto);
+  @include padding(null $gutter);
+  max-width: 500px;
+  position: relative;
+}
+
+.welcome-message {
+  @include clearfix();
+  @include padding($base-spacing null);
+  background-color: $blue;
+  margin-bottom: 1.5rem;
+  text-align: center;
+}
+
+.welcome-message-title {
+  font-family: "helvetica Neue", "Helvetica", sans-serif;
+  font-weight: 200;
+  line-height: $base-line-height;
+}
+
+// ============================================
+
+body {
+  color: $darkerblue;
+  line-height: $base-line-height;
+  text-align: center;
+}
+
+h3 {
+  margin: 1em;
+}
+
+pre {
+  line-height: 1.6em;
+}
+
+a,
+a:visited {
+  color: inherit;
+}
+
+.code {
+  @include outer-container();
+  background-color: $lightblue;
+  padding: 0.5em;
+  text-align: center;
+}
+
+.page-section {
+  @include outer-container;
+  background-color: $lightblue;
+  line-height: $base-line-height * 1.5;
+  margin-bottom: 3em;
+  text-align: center;
+}
+
+.page-section-inner {
+  background-color: $blue;
+  font-size: 0.8em;
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  margin-bottom: 2em;
+  padding: 1em;
+  text-align: left;
+}
+
+// Zero
+// ============================================
+
+.zero-alpha,
+.zero-beta {
+  @include span-columns(6);
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  padding: 1em;
+  text-align: left;
+}
+
+// First
+// ============================================
+
+.first-alpha {
+  @include span-columns(1);
+}
+
+.first-beta {
+  @include span-columns(11);
+}
+
+.first-gamma {
+  @include span-columns(2);
+}
+
+.first-delta {
+  @include span-columns(10);
+}
+
+.first-epsilon {
+  @include span-columns(3);
+}
+
+.first-zeta {
+  @include span-columns(9);
+}
+
+.first-eta {
+  @include span-columns(4);
+}
+
+.first-theta {
+  @include span-columns(8);
+}
+
+.first-iota {
+  @include span-columns(5);
+}
+
+.first-kappa {
+  @include span-columns(7);
+}
+
+.first-lambda {
+  @include span-columns(6);
+}
+
+.first-mu {
+  @include span-columns(6);
+}
+
+.first-beta,
+.first-delta,
+.first-zeta,
+.first-theta,
+.first-kappa,
+.first-mu {
+  @include omega;
+}
+
+// Second
+// ============================================
+
+.second-alpha {
+  @include span-columns(4);
+}
+
+.second-beta {
+  @include span-columns(8);
+  line-height: $base-line-height * 2;
+  padding: 0;
+  text-align: center;
+}
+
+.second-beta-alpha,
+.second-beta-beta {
+  @include span-columns(4 of 8);
+  background-color: $lightblue;
+  height: 84%;
+  padding: 1em;
+}
+
+// Third
+// ===================================
+
+.third-dummy {
+  border-bottom: 0;
+  height: 5.5em;
+  text-align: center;
+}
+
+.third-alpha {
+  @include fill-parent();
+  @include row(table);
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.third-alpha-alpha {
+  @include span-columns(4);
+  border-right: 1px solid rgba(255,255,255, 0.5);
```

Comment:
Color literals like `rgba(255, 255, 255, 0.5)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space